lib/mm_network.pm

```diff
@@ -84,6 +105,7 @@ sub configure_static_dns {
     my $servers = join(" ", @{$conf->{nameserver}});
     assert_script_run("sed -i -e 's|^NETCONFIG_DNS_STATIC_SERVERS=.*|NETCONFIG_DNS_STATIC_SERVERS=\"$servers\"|' /etc/sysconfig/network/config");
```

Comment:
This isn't used with NetworkManager, the DNS configuration has to be performed with `nmcli` as well